bindings/python/mata.pyx

```diff
@@ -149,16 +149,19 @@ cdef class Nfa:
     #  potentially create some kind of Factory/Allocator/Pool class, that would take care of management of the pointers
     #  to optimize the shared pointers away if we find that the overhead is becoming too significant to ignore.
     cdef shared_ptr[mata.CNfa] thisptr
-    cdef alphabet
 
-    def __cinit__(self, state_number = 0, alphabet = None):
-        """Constructor of the NFA
+    def __cinit__(self, state_number = 0, Alphabet alphabet = None):
+        """
+        Constructor of the NFA.
```

Comment:
Just small suggestion: the long python comments are written as above (`"""text` not `"""\ntext`), this is due to many subsequent tools, that uses the comments e.g. to generate documentation or when you access the docstring in interpret. Also it is good to keep the first line of comment short (so it can fit in terminal, add empty line and them longer explanation of the comment.

Reply:
From [PEP 257](https://peps.python.org/pep-0257/#multi-line-docstrings):
> The summary line may be on the same line as the opening quotes or on the next line.

By PEP 257, both approaches are acceptable. That the tools working with the docstrings do not follow the PEP is their wrong implementation of the PEP 257. However, because we want to use these tools, we should consider whether our currently used tools support both formats and, as the case may be, convert all docstring to the supported format.

Reply:
To be honest, I can't seem to recall ever seeing the other "acceptable" usage :). Maybe it is my pet peeve.

Reply:
I have seen both used interchangeably. Nevertheless, as of now, we are the only developers of the Python binding. Therefore, we can agree on one format and stick with it everywhere. Shall we use `"""text`, then? I can rewrite all docstrings to adhere to this convention.

Reply:
Fixed.